crates/kubelet/src/config.rs

```diff
@@ -260,3 +282,12 @@ fn is_same_ip_family(first: &IpAddr, second: &IpAddr) -> bool {
         IpAddr::V6(_) => second.is_ipv6(),
     }
 }
+
+fn split_one_label(in_string: &str) -> Result<(String, String), String> {
+    let mut splitter = in_string.splitn(2, '=');
+    let key = splitter.next().unwrap();
+    match splitter.next() {
+        Some(val) => Ok((key.to_string(), val.to_string())),
+        None => Err("empty label".to_string()),
```

Comment:
Label values in other resources are allowed to be empty.  Is that not true of nodes?

Reply:
Yup, labels can be empty. I adjusted the code to take it into account.